cmd/exec.go

```diff
@@ -187,10 +199,20 @@ func RunCommand(subcommand string, withEnvironment bool, withMonitoring bool) in
 			endTime := makeStamp()
 			duration := endTime - startTime
 			exitCode := 0
+
 			if err == nil {
 				if withMonitoring {
 					monitoringWaitGroup.Add(1)
-					go sendPing("complete", monitorCode, string(outputForPing), formattedStartTime, endTime, &duration, &exitCode, &monitoringWaitGroup)
+					go sendPing("complete", monitorCode, string(outputForPing), series, endTime, &duration, &exitCode, metrics, &monitoringWaitGroup)
+					monitoringWaitGroup.Add(1)
+					go func(wg *sync.WaitGroup) {
+						outputForLogs := gatherOutput(tempFile, false)
+						_, err = shipLogData(monitorCode, series, string(outputForLogs))
+						if err != nil {
+							fmt.Printf("%v", err)
```

Comment:
switch this to a log() function call and do the same on the other block. 

Also since this inline goroutine is like 5+ LOC and used twice, i would probably just give it a name and call it